cluster/CHAPTER.rst

```diff
@@ -1269,11 +1248,9 @@ When something *does* break, all of the debugging support becomes critical: if t
 
 I performed some manual fuzz testing of cluster during development, but a full fuzz-testing infrastructure is beyond the scope of this project.
 
-Implementation Challenges
-=========================
+## Implementation Challenges
 
-Follow the Leader
```

Comment:
The "Implementation Challenges" section only has one subsection, "Follow the Leader" -- would it be possible to move the Follow the Leader section up into an earlier part of the chapter? Is there somewhere else it would make sense?